src/distribution/gamma.rs

```diff
@@ -146,6 +147,51 @@ impl ContinuousCDF<f64, f64> for Gamma {
             gamma::gamma_ur(self.shape, x * self.rate)
         }
     }
+
+    fn inverse_cdf(&self, p: f64) -> f64 {
+        const MAX_ITERS: (u16, u16) = (8, 4);
```

Comment:
Is there a special reason for defining the number of iterations like this? It's probably more natural to just
```rs
for _ in 0..8 {
```

since the complexity is pretty low

Reply:
I thought,
> 8 and 4 sound like nice numbers, what does that give me. 8 bisection is 2 digits in $p$ and NR is quadratic convergence, so 4 gives 16 digits, lucky me. Hmm, it's helpful to name things, so `const` it is. 

Then I thought,
> I don't want to name two close but separate things, tuple it is. 

I.e. I would be fine changing it to literal.

However, since my attention is on it, I think I might instead do the initial bracketing a little differently and try some math to back up the number of iterations I select.

Reply:
I see. It's not a problem either way, I just wanted to know why you used that const tuple there.